sdcm/results_analyze/__init__.py

```diff
@@ -302,27 +329,32 @@ def check_regression(self, test_id, data, is_gce=False, node_benchmarks=None):
             test_version = ''
             build_id = ''
 
-        last_error_events, error_events_summary = self.get_events(event_severity=[Severity.ERROR.name])
-        last_critical_events, critical_events_summary = self.get_events(event_severity=[Severity.CRITICAL.name])
-        last_events = {Severity.ERROR.name: last_error_events[Severity.ERROR.name][:100],
-                       Severity.CRITICAL.name: last_critical_events[Severity.CRITICAL.name][:100]}
-        events_summary = {
-            Severity.ERROR.name: error_events_summary[Severity.ERROR.name],
-            Severity.CRITICAL.name: critical_events_summary[Severity.CRITICAL.name]
-        }
+        # last_error_events, error_events_summary = self.get_events(event_severity=[Severity.ERROR.name])
```

Comment:
why were these events commented out? if not needed anymore, they should be removed